Capfile

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+# Load DSL and set up stages
+require 'capistrano/setup'
+
+# Include default deployment tasks
+require 'capistrano/deploy'
+
+# Load the SCM plugin appropriate to your project:
+#
+# require "capistrano/scm/hg"
+# install_plugin Capistrano::SCM::Hg
+# or
+# require "capistrano/scm/svn"
+# install_plugin Capistrano::SCM::Svn
+# or
+require 'capistrano/scm/git'
+install_plugin Capistrano::SCM::Git
+
+# Include tasks from other gems included in your Gemfile
+#
+# For documentation on these, see for example:
+#
+#   https://github.com/capistrano/rvm
+#   https://github.com/capistrano/rbenv
+#   https://github.com/capistrano/chruby
+#   https://github.com/capistrano/bundler
+#   https://github.com/capistrano/rails
+#   https://github.com/capistrano/passenger
+#
+# require "capistrano/rvm"
+require "capistrano/rbenv"
+# require "capistrano/chruby"
+require "capistrano/bundler"
+# require "capistrano/rails/assets"
+# require "capistrano/rails/migrations"
+# require "capistrano/passenger"
+# require 'capistrano/rails'
+
+# Web server plugins
+require 'capistrano/puma'
+require 'capistrano/puma/workers'
+
+
+install_plugin Capistrano::Puma  # Default puma tasks
```

Comment:
`install_plugin Capistrano::Puma::Systemd` is missing

From https://github.com/seuros/capistrano-puma?tab=readme-ov-file#usage